### ISSUE TITLE
Inquiry about ViT-Tiny (Linear) Training Strategy

### ISSUE THREAD

**Issue by uwGZQ** [aimagelab/CoDE#1]
Hello, authors,

Thank you for your valuable work! I have a few questions regarding the training process of the ViT-T (Linear) model in Table 2. Could you please clarify if you added an nn.Linear(model.head.in_features, 1) layer followed by nn.Sigmoid() on top of the ViT-Tiny? Additionally, did you fine-tune all the parameters, or did you keep the pretrained parameters of the ViT backbone fixed and only train the linear classifier?

**Comment by federico1-creator** [comment]
Hi @uwGZQ, thank you for your interest in our project!

We trained all the parameters of the ViT-Tiny model from scratch.
In addition, for the linear layer, we train a logistic regression on the features extracted by ViT.

Technically, we use sklearn with the following hyperparameters:
`
from sklearn.linear_model import LogisticRegression
classifier = LogisticRegression(random_state=args.seed, C=0.316, max_iter=args.epochs, verbose=1, class_weight='balanced')
`
